velox/common/base/tests/SimdUtilTest.cpp

```diff
@@ -349,4 +349,22 @@ TEST_F(SimdUtilTest, Batch64_memory) {
   EXPECT_EQ(data[1], 2);
 }
 
+template <typename T>
+void validateReinterpretBatch() {
+  using U = typename std::make_unsigned<T>::type;
+  auto result = simd::reinterpretBatch<U>(simd::setAll<T>(-1));
+  auto result2 = simd::reinterpretBatch<T>(simd::setAll<T>(-1));
+  for (int i = 0; i < result.size; ++i) {
+    EXPECT_EQ(result.get(i), std::numeric_limits<U>::max());
```

Comment:
use ASSERT_EQ to stop at the first failed iteration to avoid generating too many error logs?